sample-app/package.json

```diff
@@ -173,5 +187,10 @@
     "presets": [
       "react-app"
     ]
+  },
+  "devDependencies": {
+    "autoprefixer": "^9.8.8",
+    "postcss": "^7.0.39",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.17"
```

Comment:
do we need the npm prefix? I've never seen this one before

Reply:
I tried removing it, but the site didn't complile. I think it's needed because I'm using the compatability build rather than the the tailwindcss repo itself, and it needs to know to look at the other repo. I was seeing an error with the regular tailwind repo, and it said it's likely to happen if a toolchain hasn't been updated to use postcss 8, and based on my testing, it didn't look like our version of create-react-app worked with it. (See https://tailwindcss.com/docs/installation#post-css-7-compatibility-build)

Reply:
is there anything wrong with using postcss 8?

Reply:
The version of create-react-app that we ejected at doesn't support it. We would need to bump several dependencies including 'css-loader', 'webpack', 'terser-webpack-plugin' and potentially more, and I'd rather not go down that rabbit hole since long term we are looking to un-eject the app. Potentially a newer version of create react app supports it, however I wasn't able to determine that by quickly looking at the Github repo or the release notes

Reply:
I can't read